scenarios/resource-creation/basic/AIStudioHubAndProject.ipynb

```diff
@@ -35,7 +35,9 @@
    "cell_type": "code",
    "execution_count": null,
    "id": "84916e91",
-   "metadata": {},
+   "metadata": {
+    "name": "subscription"
```

Comment:
Could "name" be changed to be more descriptive (e.g. `Azure-Docs-Code-Sample`)?

Reply:
@kdestin Unfortunately no. The docfx publishing system we use for docs.microsoft.com looks for the "name" tag. It'd take a feature request to the platform team to change it. The value of that tag has to be unique and can be whatever.